### ISSUE TITLE
We dont have a published 'code of conduct' document

### ISSUE THREAD

**Issue by psh** [commons-app/apps-android-commons#1463]
I took a look at our Github “Community Profile” (https://github.com/commons-app/apps-android-commons/community) which highlights that our project doesn’t have a published code of conduct.  Digging a little deeper there’s a suggestion that the “Contributor Covenant” (see: https://www.contributor-covenant.org/) might be a good starting point.

I read through it and it seems to have been adopted by a large number of solid open-source projects and I don’t see anything I’d object to.  How does everyone feel about adopting it for our project?

**Comment by misaochan** [comment]
Good point @psh ! The "Contributor Covenant" looks good to me. I did wonder if we would be expected to use the Wikimedia code of conduct instead, but that would be much less specific and at any rate it isn't an explicit requirement for us.

**Comment by psh** [comment]
Can someone with contacts at WMF reach out to them to see if the "Contributor Covenant" is compatible with the WMF guidelines before we jump in?

**Comment by chrisdebian** [comment]
@psh Hi, Paul. Just trying to help prioritize what development work gets done next. Would you be able to have a look at this https://github.com/commons-app/apps-android-commons/issues/6034 and potentially add your thoughts/ priorities to the spreadsheet?

Thanks,

Chris.
